chat-commands.js

```diff
@@ -3409,6 +3409,21 @@ exports.commands = {
 				group: targetUser.group,
 				rooms: roomList,
 			};
+			if (targetUser in room.users && user.can('addhtml', null, room)) {
```

Comment:
That's not necessarily a bad thing? Challenges probably don't need to be super secret.